app/scripts/controller-init/types.ts

```diff
@@ -30,7 +30,11 @@ export type ControllerByName = {
  * e.g. `{ TransactionController: { transactions: [] } }`.
  */
 export type ControllerPersistedState = Partial<{
-  [name in ControllerName]: Partial<ControllerByName[name]['state']>;
+  [name in ControllerName]: Partial<
```

Comment:
assetsContractController does not extend `BaseController` , i'm updating this to make `state` optional